tests/aws/services/cloudformation/api/test_nested_stacks.py

```diff
@@ -81,7 +82,7 @@ def test_nested_stack_output_refs(deploy_cfn_template, s3_create_bucket, aws_cli
     assert f"{nested_bucket_name}-suffix" == result.outputs["CustomOutput"]
 
 
-@pytest.mark.skip(reason="Nested stacks don't work properly")
+@pytest.mark.skip(reason="Nested rtacks don't work properly")
```

Comment:
```suggestion
@pytest.mark.skip(reason="Nested stacks don't work properly")
```

Reply:
An error from my part but thanks to it I found that we can delete the skip